docs/products/metrics/howto/storage-overview-page.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Storage overview for Aiven for Metrics
+sidebar_label: Storage overview page
+---
+Get complete insights into your storage usage with Aiven for Metrics. This overview page offers data that helps you better manage and understand your object storage usage and associated costs. Access important metrics such as current and forecasted billing, object storage consumption, and settings for data retention and storage threshold alerts.
+
+## Access the Storage Overview page
+
+1. Log into the [Aiven console](https://console.aiven.io/), select your project, and
+   select your Aiven for Metrics service.
+1. Click **Storage** in the sidebar.
+
+
+## Key storage insights
+
+- **Current billing expenses**: Shows the current cost incurred for the storage used.
+- **Forecasted monthly cost**: Provides an estimate of the expected cost for the
+  upcoming month based on usage.
+- **Object storage used**: Indicates the amount of storage used in MiB.
```

Comment:
I think it could not only be MiB right? It depends on the customer's used storage so better to remove mention of a concrete unit?

Reply:
Thanks! Removed the unit.